### DISCUSSION TITLE
Tag input on Hunternif fork

### DISCUSSION THREAD

**Discussion by dyedfire** [rr-/szurubooru#573] [Q&A]
Hello,

I've been using Hunternif's fork of Szurubooru and I ran into an issue involving adding tags via the clipboard. Before I switched to Hunternif's fork, I was able to paste a list of tags that I copied from one post and paste it into the tag control box like such:
```
Example1 
Example2_test 
Example_3test 
```

This would even populate "implications" if any were on the list. However, the fork does not allow this and presents the error "Name too long" or "Name must satisfy regex '^\\S+$'."

I'm not sure if anyone has encountered this issue or what has changed between the two, but if anyone could point me in the right direction, it would be appreciated.

**Comment by parallax4** [comment]
@Hunternif

**Comment by Hunternif** [comment]
Must be a bug. I have not checked to properly integrate the new baseline tag functionality into mine.
